projects/admin/src/app/routes/route-tool.service.ts

```diff
@@ -144,7 +144,7 @@ export class RouteToolService {
    */
   canSystemLibrarian(message: string = ''): Observable<ActionStatus> {
     return of(
-      { can: this._userService.hasRole('system_librarian'), message }
+      { can: this._userService.user.isLibrarian, message }
```

Comment:
librarian <> sys_lib ?

Reply:
I added a proxy on the role system_librarian
done